### DISCUSSION TITLE
Allow to browse files from the web UI

### DISCUSSION THREAD

**Discussion by ThePlenkov** [verdaccio/verdaccio#4796] [Ideas]
Hi! I'd like to propose to add the feature to browse files as it works in npmjs on the code tab;  https://www.npmjs.com/package/verdaccio?activeTab=code

This would give us following benefits:
- ability to verify that the package contains right files
-  ability to see files sizes, so it helps to analyse the package size and what it consists of
- ability to view certain files

In understand that technically it is stored as an archive, so may be unarchaving on the fly or activating an extra storage in unarchavied format would be the option

It makes sense to make this feature configurable ( in case if it takes extra storage let's say )

**Comment by HawtinZeng** [comment]
4. The ability to show all cached packages.
5. The ability to transfer some cached packages into another machine which is offline.
